classes/class-excludes.php

```diff
@@ -195,10 +195,6 @@ function( $exclude ) {
 	 */
 	public function is_file_excluded( \SplFileInfo $file ) {
 
-		if ( ! $file->isFile() ) {
```

Comment:
The `$files` array in https://github.com/humanmade/backupwordpress/blob/master/admin/schedule-form-excludes.php#L233 contains both directories and files so the new `is_file_excluded` method will need to handle both too. Perhaps renaming the method to `is_path_excluded` would make more sense.